core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -125,41 +124,12 @@ public class GenericContainer<SELF extends GenericContainer<SELF>>
     /*
      * Default settings
      */
-    @NonNull
-    private LinkedHashSet<Integer> exposedPorts = new LinkedHashSet<>();
-
-    @NonNull
-    private List<String> portBindings = new ArrayList<>();
-
     @NonNull
     private List<String> extraHosts = new ArrayList<>();
 
-    @NonNull
-    private String networkMode;
-
-    @Nullable
-    private Network network;
-
-    @NonNull
-    private List<String> networkAliases = new ArrayList<>(Arrays.asList("tc-" + Base58.randomString(8)));
```

Comment:
Selenium tests will break when creating VncContainer because there is no network